README.md

```diff
@@ -45,4 +45,114 @@ sanitized_promise.then(function(sanitized_string) {
 
 ## Contributing
 
+### Commits
+
 Use `npm run commit` when you want to commit a change.
+
+### Releases
+
+This project uses [GitHub actions](https://docs.github.com/en/actions/reference) 
+and [semantic-release](https://github.com/semantic-release/semantic-release) for creating releases.
+
+#### Release Candidates
+
+To make a (temporary) release candidate, you can use the following commands:
+
+```bash
+# Create a new rc from the latest/remote develop
+npm run release-candidate
+```
+
+or 
+
+```bash
+# Create a new rc from a specific branch
+npm run release-candidate -- feature/SKED-XXXX
+```
+
+That command will make a new `rc` branch (locally and remotely) on which `semantic-release` is configured
+to create a new release candidate (see `.releaserc`).
+
+#### Final Releases
+
+Since `semantic-release` is currently configured to run on any `push`'es to `master`,
+creating and merging a GitHub Pull Request into `master` will trigger a new release automatically.
+
+Typically we do this via a temporary `release/next` or `release/SKED-XXXX` branch and creating a PR via GitHub UI.
+
+### Stack
+
+This repostiory uses [TSDX](https://tsdx.io/) for development.
+
+## Commands
+
+TSDX scaffolds your new library inside `/src`.
+
+To run TSDX, use:
+
+```bash
+npm start # or yarn start
+```
+
+This builds to `/dist` and runs the project in watch mode so any edits you save inside `src` causes a rebuild to `/dist`.
+
+To do a one-off build, use `npm run build` or `yarn build`.
+
+To run tests, use `npm test` or `yarn test`.
+
+### Jest
+
+Jest tests are set up to run with `npm test` or `yarn test`.
+
+### Bundle Analysis
+
+[`size-limit`](https://github.com/ai/size-limit) is set up to calculate the real cost of your library with `npm run size` and visualize the bundle with `npm run analyze`.
+
+### Rollup
+
+TSDX uses [Rollup](https://rollupjs.org) as a bundler and generates multiple rollup configs for various module formats and build settings. See [Optimizations](#optimizations) for details.
+
+### TypeScript
+
+`tsconfig.json` is set up to interpret `dom` and `esnext` types, as well as `react` for `jsx`. Adjust according to your needs.
```

Comment:
Maybe we can remove references to react and jsx (and about the styles at the bottom of the file) because it's not really related to this package.